sos/plugins/libvirt.py

```diff
@@ -70,4 +71,12 @@ def postproc(self):
                     r"\1******\3"
                 )
 
+        protect_keys = [
+            "vnc_password", "spice_password"
+        ]
+
+        regexp = r"((?m)^\s*(%s)\s*=\s*)(.*)" % "|".join(protect_keys)
+        self.do_path_regex_sub("/etc/libvirt/qemu.conf",
+                               regexp, r"\1*********")
+
```

Comment:
There's no need to use `do_path_regex_sub()` here - that's only for when you have multiple files, whose paths match some regular expresion, and that you want to apply substitution for all files with a single call.

Since you only have a single file here you should use `do_file_sub()`, e.g.:

```python
        self.do_file_sub(
            "/etc/ovirt-engine/redhatsupportplugin.conf",
            r"proxyPassword=(.*)",
            r"proxyPassword=********"
        )
```